install_config/upgrading/automated_upgrades.adoc

```diff
@@ -146,18 +146,32 @@ channel and enable the 3.4 channel on each master and node host:
 # yum update atomic-openshift-utils
 ----
 
-. Install or update to the following latest available **-excluder* packages on
-each RHEL 7 system, which helps ensure your systems stay on the correct versions
-of *atomic-openshift* and *docker* packages when you are not trying to upgrade,
-according to the {product-title} version:
+. Install or update to the following latest available *atomic-openshift-excluder*
+package on each RHEL 7 system, which helps ensure your systems stay on the
+correct versions of *atomic-openshift* packages when you are not trying to
+upgrade, according to the {product-title} version:
```

Comment:
> This is not clear to me.
> "...update to the following latest available... package" Not sure what "following latest" means.

@mburke5678 Ah yeah, that made more sense when it was the previous "update to the following latest available *-excluder packages" and then two different package were listed after.

> "...when you are not trying to upgrade..." This seems to indicate that I am not trying to update the packages. If so, it seems redundant and a bit confusing.
> "s/"which helps ensure your systems stay on the correct versions of atomic-openshift packages for your {product-title} version." (??)

It's explaining what the packages are for in general. This is somewhat a copy/paste from the similar install instruction step, but we want to make sure that they're on the latest version of these packages before they do start a cluster upgrade, and also just to make sure they're installed (if for some reason they weren't) because otherwise later steps won't make sense.